### PR TITLE
feature(main): fix already installed buildah

### PR DESCRIPTION
**Pull request by cuisongliu**
Signed-off-by: cuisongliu <cuisongliu@qq.com>

### CODE REVIEW

File: pkg/image/buildah.go [hunk 1]
```diff
@@ -37,9 +42,43 @@ func initBuildah() (bool, error) {
...
+		versionJSON, _ := exec.RunBashCmd("buildah version")
+		var bVersion BuildahVersion
+		_ = json.Unmarshal([]byte(versionJSON), &bVersion)
+		logger.Debug("buildah version: %+v", bVersion)
+		onceBuildahVersion = bVersion
+	})
+	if onceBuildahVersion.GitCommit == "bc5080cc" {
+		return true, nil
+	}
+	return false, errors.New("system buildah is not sealos buildah,please uninstall buildah retry it")
```
Comment (fanux) [inline pkg/image/buildah.go:65]: What means "system buildah is not sealos buildah...."

**Comment by fanux** [comment]
What can cause problems if buildah is already installed？

**Comment by fanux** [comment]
What can cause problems if buildah is already installed？

**Comment by cuisongliu** [comment]
> What can cause problems if buildah is already installed？

It may cause the sealos build to be unsuccessful, which is not the same version as the buildah we use, and the parameters are different.

**Comment by cuisongliu** [comment]
fix https://github.com/labring/sealos/issues/1756 , delete buildah check

File: pkg/image/buildah.go (hunk 1 above)
Comment (zzjin) [inline pkg/image/buildah.go:65]:
```suggestion
	return false, errors.New("System's buildah is not match sealos buildah requirements, please uninstall system's buildah, and retry")
```

**Review by zzjin — APPROVED** [review]

File: pkg/image/buildah.go [hunk 2]
```diff
@@ -37,9 +42,43 @@ func initBuildah() (bool, error) {
...
+		versionJSON, _ := exec.RunBashCmd("buildah version")
+		var bVersion BuildahVersion
+		_ = json.Unmarshal([]byte(versionJSON), &bVersion)
+		logger.Debug("buildah version: %+v", bVersion)
+		onceBuildahVersion = bVersion
+	})
+	if onceBuildahVersion.GitCommit == "bc5080cc" {
+		return true, nil
+	}
+	return false, errors.New("System's buildah is not match sealos buildah requirements, please uninstall system's buildah, and retry")
```
Comment (fanux) [inline pkg/image/buildah.go:65]: But how to fix this issue, user don't know the gitcommit ID，how to reinstall.

**Comment by fanux** [comment]
1. Users don't know which commitID to download 2. Why other versions of buildah don't work, because our integrated SDK is not new enough, is there a way to be compatible with other versions of buildah 3. The fundamental solution is to not rely on the buildah binary at all

**Pull request merged**